### ISSUE TITLE
Features Suggestions

### ISSUE THREAD

**Issue by Keplyx** [Timocop/BasicPawn#14]
Hi, I thought of some features that could make the editor a little better:

- It would be cool to have a Line Wrap option, because it is sometimes annoying to scroll horizontally when the window is smaller and some lines do not fit the screen.

- An other useful option would be to automatically open project files when opening a project

And last, I wanted to report that the Document Minimap option is making the editor very slow with large files (when creating a new line, or switching files), at least when using BasicPawn under wine 2.0.4. It is impossible to use on files with over a 1000 lines.

**Comment by Timocop** [comment]
You mean like Word Warp? And since i optimized tab loadings i could do the second one too.

And yea you're right, i just tested it with 9k lines source with Doc Minimap enabled under Ubuntu Wine 2.0.2. It hangs very badly. Disable Document Minimap for now (View > Document Minimap) because i currently have no fix for that.

**Comment by Keplyx** [comment]
Yes word warp, when activated it breaks lines that do not fit the screen.

I'm don't really need the document minimap anyway, but just wanted to report it.

**Comment by Timocop** [comment]
Doesn't seem like the old ICSharpCode TextEditor supports Word Warp :/
